functions.php

```diff
@@ -120,7 +127,7 @@ function _s_scripts() {
 /**
  * Implement the Custom Header feature.
  */
-//require get_template_directory() . '/inc/custom-header.php';
+require get_template_directory() . '/inc/custom-header.php';
```

Comment:
I also think that uncommenting this is totally fine as well. It always struck me as weird that _s had commented code for functionality like featured post thumbnails and the custom header. Thanks for handling this.